### PR TITLE
tests: wait for all traces to be send

### PR DESCRIPTION
**Pull request by jirikuncar**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@ def load_fixture(fixture)
   end
 
   config.after(:suite) do
-    Datadog.tracer.shutdown!
```
Comment (zippolyte) [inline spec/spec_helper.rb:110]: no need to shutdown anymore ? or should it still be called after ?

Comment (jirikuncar) [reply spec/spec_helper.rb:110]: yes, it will be call at exit

Comment (jirikuncar) [reply spec/spec_helper.rb:110]: we need to make sure that the worker buffer is empty instead.

**Review by zippolyte — APPROVED** [review]

**Pull request merged**